### PR TITLE
Support multipolygon geojson label stores

### LINKED ISSUE #330: Add support for multipolygons
Enable RV to handle geojson label stores that are encoded as multipolygons

### PR DESCRIPTION
**Pull request by simonkassel**
## Overview

This PR enables raster vision to handle geojson files encoded as multipolygons when creating training chips for object detection. 

### Checklist

- [x] Ran scripts/format_code and commited any changes
- [x] Documentation updated if needed
- [x] PR has a name that won't get you publicly shamed for vagueness

### Notes

This has been tested by creating training chips from the spacenet dataset.

## Testing Instructions

* Run unit tests `python -m rastervision.label_stores.object_detection_geojson_file_test`

Closes #330

### CODE REVIEW

File: src/rastervision/label_stores/object_detection_geojson_file_test.py [hunk 1]
```diff
@@ -79,6 +127,33 @@ def tearDown(self):
         self.mock_s3.stop()
         self.temp_dir.cleanup()
 
+    def test_multipolygon_geojson_to_labels(self):
+        geojson = add_classes_to_geojson(self.multipolygon_geojson_dict,
+                                         self.class_map)
+        labels = geojson_to_labels(geojson, self.crs_transformer)
+        label_coordinates = labels.boxlist.get()
+
+        expected_box_coordinates = np.array(
```
Comment (lewfish) [inline src/rastervision/label_stores/object_detection_geojson_file_test.py]: This only tests that the boxes are correct, but you should also test that the classes and scores in the labels are correct. To make this easier, you should construct an `expected_labels` object and then use the `ObjectDetectionLabels.assert_equal` method.

File: src/rastervision/label_stores/object_detection_geojson_file_test.py [hunk 2]
```diff
@@ -79,6 +127,33 @@ def tearDown(self):
...
+        label_coordinates = labels.boxlist.get()
+
+        expected_box_coordinates = np.array([[0., 0., 2., 2.],
+                                             [2., 2., 4., 4.]])
+        self.assertTrue(
+            np.array_equal(label_coordinates, expected_box_coordinates))
+
+    def test_read_invalid_geometry_type(self):
+        with self.assertRaises(Exception):
+            geojson = add_classes_to_geojson(self.linestring_geojson_dict,
```
Comment (lewfish) [inline src/rastervision/label_stores/object_detection_geojson_file_test.py]: Nice job testing the error condition.

File: src/rastervision/label_stores/object_detection_geojson_file_test.py (hunk 1 above)
Comment (simonkassel) [reply src/rastervision/label_stores/object_detection_geojson_file_test.py]: that makes sense, will do

**Comment by lewfish** [comment]
Your changes look good but the style_tests are failing.

**Pull request merged**